solana/modules/token_bridge/program/src/messages.rs

```diff
@@ -386,6 +372,7 @@ impl DeserializeGovernancePayload for GovernancePayloadUpgrade {
 }
 
 #[cfg(feature = "no-entrypoint")]
+#[allow(unused_imports)]
```

Comment:
Why not remove the unused imports here as well?

Reply:
This was a weird case, the compiler was raising a lot of spurious unused import warnings in the test code.